app/views/step_by_step/show.html.erb

```diff
@@ -116,16 +116,14 @@
             </h2>
           </div>
           <div class="app-step-nav__panel js-panel js-hidden" id="step-panel-select-a-training-partner-5">
-            <p class="app-step-nav__paragraph">If you choose to use a training provider, your induction tutor needs to
-              select from the 6 available options as soon as possible.</p>
+            <p class="app-step-nav__paragraph">If you choose to use a training provider, your induction tutor needs to select from the 6 available options as soon as possible.</p>
             <ol class="app-step-nav__list " data-length="2">
-              <li class="app-step-nav__list-item js-list-item ">
```

Comment:
I'm curious why we needed an `app-step-nav__list` here and not just `app-step-nav__paragraph` ?